sources/packages/api/src/route-controllers/designation-agreement/designation-agreement.controller.ts

```diff
@@ -114,4 +127,32 @@ export class DesignationAgreementController {
       ),
     } as GetDesignationAgreementDto;
   }
+
+  /**
+   * Get the list of all the designations that belongs to
+   * the institution user currently authenticated.
+   * @param userToken user authentication token.
+   * @returns the list of all the designations that
+   * belongs to one the institution.
```

Comment:
Did you mean to write "belongs to one of the institutions" here ?

Reply:
It is supposed to be "the list of all the designations that belongs to one institution."
I updated it 👍